astropy/time/core.py

```diff
@@ -789,6 +789,9 @@ def __setitem__(self, item, value):
         self._time.jd1[item] = value._time.jd1
         self._time.jd2[item] = value._time.jd2
 
+    def __hash__(self):
```

Comment:
Remove if not needed.

Reply:
This hash is also incomplete (should include scale and location), so either adjust or remove.